packages/next/build/webpack/loaders/next-babel-loader.js

```diff
@@ -34,6 +34,8 @@ module.exports = babelLoader.custom(babel => {
             cacheDirectory: join(opts.distDir, 'cache', 'next-babel-loader'),
             cacheIdentifier:
                 cacheKey +
+                // @TODO: this point of variability doesn't seem to be accounted for?
+                // (opts.isServer ? '-server' : '') +
```

Comment:
@timneutkens @Timer I haven't found a way to check that this is an issue/non-issue yet. Do you know off-hand if the client and server webpack passes share a Babel cache key?

Reply:
Off hand, they do share a cache key. We'll need to add the server delimiter here.